dvc/remote/ssh/__init__.py

```diff
@@ -38,15 +39,20 @@ def __init__(self, repo, config):
 
         parsed = urlparse(self.url)
         self.host = parsed.hostname
+
+        user_ssh_config = self._load_user_ssh_config(self.host)
```

Comment:
Btw, if we are looking up `Host` in the ssh config, shouldn't we use `HostName` after that?

Reply:
You are right, we should use `HostName` from ssh config if it exists.